src/main/java/com/faforever/client/game/PlayerCardTooltipController.java

```diff
@@ -37,10 +37,10 @@ public void setPlayer(Player player, int rating) {
     }
     countryFlagService.loadCountryFlag(player.getCountry()).ifPresent(image -> countryImageView.setImage(image));
     
-    if (!StringUtils.isEmpty(player.getCountry())) {
-      final Tooltip countryTooltip = new Tooltip(i18n.getCountryNameLocalized(player.getCountry()));
-      //countryTooltip.textProperty().bind(player.countryProperty());
-      Tooltip.install(playerInfo, countryTooltip); //countryImageView won't work because part of label.
+    if (!StringUtils.isEmpty(player.getCountry())) { // empty (non-null) string -> empty tooltip
```

Comment:
what if there is no country can that happen(null)?

Reply:
I‘m not sure, but it doesn’t matter here. A tooltip shall only be created for non-empty strings.